server/lib/scene/scene.actions.js

```diff
@@ -30,92 +30,56 @@ const actionsFunc = {
     return self.device.setValue(device, deviceFeature, action.value);
   },
   [ACTIONS.LIGHT.TURN_ON]: async (self, action, scope) => {
-    await Promise.map(action.devices, async (deviceSelector) => {
-      try {
-        const device = self.stateManager.get('device', deviceSelector);
-        const deviceFeature = getDeviceFeature(
-          device,
-          DEVICE_FEATURE_CATEGORIES.LIGHT,
-          DEVICE_FEATURE_TYPES.LIGHT.BINARY,
-        );
+    await Promise.map(action.device_features, async (deviceFeatureSelector) => {
+      const deviceFeature = self.stateManager.get('deviceFeature', deviceFeatureSelector);
+      if (deviceFeature) {
+        const device = self.stateManager.get('deviceById', deviceFeature.device_id);
         await self.device.setValue(device, deviceFeature, 1);
-      } catch (e) {
-        logger.warn(e);
       }
     });
   },
   [ACTIONS.LIGHT.TURN_OFF]: async (self, action, scope) => {
-    await Promise.map(action.devices, async (deviceSelector) => {
-      try {
-        const device = self.stateManager.get('device', deviceSelector);
-        const deviceFeature = getDeviceFeature(
-          device,
-          DEVICE_FEATURE_CATEGORIES.LIGHT,
-          DEVICE_FEATURE_TYPES.LIGHT.BINARY,
-        );
+    await Promise.map(action.device_features, async (deviceFeatureSelector) => {
+      const deviceFeature = self.stateManager.get('deviceFeature', deviceFeatureSelector);
+      if (deviceFeature) {
+        const device = self.stateManager.get('deviceById', deviceFeature.device_id);
         await self.device.setValue(device, deviceFeature, 0);
-      } catch (e) {
```

Comment:
Why removes all the catch ? the goal of the catch here was:

- If a light doesn't respond, we still want to shut down the other lamps, it's weird to stop the scene for this?